src/main/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImpl.java

```diff
@@ -62,6 +62,7 @@ public void anonymizeCourtCaseEntity(UserAccountEntity userAccount, CourtCaseEnt
         courtCase.getDefenceList().forEach(defenceEntity -> anonymizeDefenceEntity(userAccount, defenceEntity));
         courtCase.getProsecutorList().forEach(prosecutorEntity -> anonymizeProsecutorEntity(userAccount, prosecutorEntity));
         courtCase.getHearings().forEach(hearingEntity -> anonymizeHearingEntity(userAccount, hearingEntity));
+        eventService.getAllCourtCaseEventVersions(courtCase).forEach(eventEntity -> anonymizeEventEntity(userAccount, eventEntity));
```

Comment:
@jackmaloney  should this only take into account event versions where all the associated cases are anonymised or due to be? or is this okay just anonymising  them all regardless?

Reply:
Yeh, it's on my todo list to raise this as a separate ticket to cover this requirement. For now, we just anonymise everything.